javaslang/src/main/java/javaslang/Predicates.java

```diff
@@ -118,4 +119,23 @@ private Predicates() {
         Objects.requireNonNull(predicates, "predicates is null");
         return anyOf(predicates).negate();
     }
+
+
+    /**
+     * Creates a {@code Predicate} that tests, if an object is null
+     * @param <T> value type
+     * @return A new {@code Predicate}
+     */
+    public static <T> Predicate<T> isNull() {
+        return Objects::isNull;
+    }
+
+    /**
+     * Creates a {@code Predicate} that tests, if an object is not null
+     * @param <T> value type
+     * @return A new {@code Predicate}
+     */
+    public static <T> Predicate<T> isNotNull() {
+        return Objects::nonNull;
+    }
```

Comment:
Very nice :-)
Please move the new methods above line 72 `// -- Predicate combinators` in order we do not confuse it with the predicate combinators.
Thanks!